src/ccsr/src/client.rs

```diff
@@ -24,18 +25,26 @@ pub struct Client {
     inner: reqwest::Client,
     url: Url,
     auth: Option<Auth>,
+    /// Opaque tokens to cleanup resources associated with overrides.
+    _drop_tokens: Vec<Box<dyn Any + Send + Sync>>,
 }
 
 impl Client {
     pub(crate) fn new(
         inner: reqwest::Client,
         url: Url,
         auth: Option<Auth>,
+        tokens: Vec<Box<dyn Any + Send + Sync>>,
```

Comment:
I think of this crate as a general-purpose Confluent Schema Registry client that we could put on crates.io, if we so desired, and so IMO it's definitely not something that ought to be taking arbitrary tokens to hold on to. Seems like something that belongs in the storage crate, wherever this gets held on to.